src/ext.js

```diff
@@ -162,8 +162,9 @@ export default function ext({ translator, automationsEnabled }) {
                   ref: 'automation',
                   dropdownOnly: true,
                   options: async () => {
-                    const automations = await fetch('../api/v1/items?resourceType=automation&limit=100').then((response) =>
-                      response.json());
+                    const automations = await fetch('../api/v1/items?resourceType=automation&limit=100').then(
```

Comment:
Nitpicking here but I think a common praxis is to line break on new method calls.

```js
const automations = await fetch('../api/v1/items?resourceType=automation&limit=100')
  .then((response) => response.json());
```

Reply:
sure, lint apparently changed this